std-bits/table/src/main/java/org/enso/table/data/index/MultiValueIndex.java

```diff
@@ -182,7 +181,8 @@ public Table makeCrossTabTable(
 
     int offset = groupingColumns.length;
     for (List<Integer> name_locs : nameIndex.locs.values()) {
-      String name = nameColumn.getStorage().getItemBoxed(name_locs.get(0)).toString();
+      Object boxed = nameColumn.getStorage().getItemBoxed(name_locs.get(0));
+      String name = boxed == null ? "" : boxed.toString();
```

Comment:
Maybe good to put a TODO to remember to revise what the name should actually be here.